gitdm/models.py

```
@@ -40,28 +45,32 @@
+    is_patient = models.BooleanField(default=True)
+
     is_staff = models.BooleanField(default=False)
     is_active = models.BooleanField(default=True)
-
     date_joined = models.DateTimeField(default=timezone.now)
 
     objects = UserManager()
 
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS: list[str] = []  # when creating superuser via createsuperuser
+    REQUIRED_FIELDS: list[str] = []
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)